server/svix-server/src/main.rs

```diff
@@ -137,7 +137,7 @@ async fn main() {
 
         if let Err(e) = futures::future::try_join_all(wait_for).await {
             tracing::error!("{e}");
-            return;
+            return Ok(());
```

Comment:
Returning `Ok(())` here maintains the same behaviour as before this PR, but could instead `anyhow::bail!(e)` to exit with code 1

Reply:
Yeah, bailing would make more sense.